examples/common.py

```diff
@@ -29,9 +29,15 @@ def waiter(client):
             loop=client.loop,
             return_when=return_when)
 
+        # Get the result(s) of the completed task(s).
+        ret = [future.result() for future in done]
+
         # Cancel any events that didn't come in.
         for future in pending:
             future.cancel()
+
+        # Return list of completed event names.
+        return ret
```

Comment:
nitpick: maybe the example below should log the event name that completed, so that it can show how one would use the return value:

```
events = await wait_for('rpl_endofmotd', 'err_nomotd')
for event in events:
    logger.debug("finished connection with {} signal".format(event))
```